templates/openstack/architecture.html

```diff
@@ -0,0 +1,263 @@
+          </p>
+          <p>
+            In order to minimise its footprint, Canonical OpenStack ships with only the key features enabled by default. Additional features, such as CaaS or a Load Balancer can be enabled post-deployment with a single terminal command.
+          </p>
+        </div>
+        <hr class="is-muted" />
+        <a class="p-button--positive" href="/openstack/docs">Access supported features</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <div class="p-section--shallow">
+          <h2>OpenStack components</h2>
+        </div>
+      </div>
+      <div class="col">
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Services expose APIs and handle user requests</li>
+          <li class="p-list__item is-ticked">Processes create a service and implement atomic cloud functions</li>
+          <li class="p-list__item is-ticked">Dashboard provides a web-based user interface to services</li>
+          <li class="p-list__item is-ticked">Client provides a command-line user interface to services</li>
+          <li class="p-list__item is-ticked">Databases store various types of records created by services</li>
+          <li class="p-list__item is-ticked">Queues facilitate communication between processes within a service</li>
+        </ul>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-9 col-start-large-4">
+        <hr class="is-muted u-hide--small" />
+        <div class="p-image-container u-hide--small">
+          {{ image(url="https://assets.ubuntu.com/v1/aa31dc00-1.2.2.png",
+                    alt="",
+                    width="1846",
+                    height="1053",
+                    hi_def=True,
+                    attrs={"class": "p-image-container__image"},
+                    loading="lazy") | safe
+          }}
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>How does it work?</h2>
+      </div>
+      <div class="col">
+        <p>
+          In principle, OpenStack resembles the behaviour of leading public clouds. Users can provision resources from their own pool through a self-service portal. All requests are handled by OpenStack itself in a fully automated way. Don't believe it?
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a href="https://ubuntu.com/engage/openstack-introduction">Read the whitepaper: “An introduction to OpenStack”&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+    <div class="row p-section--shallow">
+      <div class="col-start-large-4 col-9 col-medium-6">
+        <hr class="is-muted" />
+        <div class="row">
```

Comment:
We can make this a proper ol using a [stepped detailed list ](https://vanillaframework.io/docs/patterns/lists#horizontal-stepped) to make it more semantic, we're already doing so [here](https://ubuntu.com/ai/consulting) for example.
Code would be: 
```
      <div class="col-start-large-4 col-9 col-medium-6">
        <ol class="p-stepped-list--detailed">
          <li class="p-stepped-list__item">
            <div class="row">
              <hr class="p-rule--muted u-hide--large">
              <div class="col-3 col-medium-3">
                <h3 class="p-heading--5 p-stepped-list__title">The user requests resources</h3>
              </div>
              <div class="col-6 col-medium-3">
                <p class="p-stepped-list__content">OpenStack enables on-demand resource provisioning. This means that users can request them at any time through the client, dashboard or APIs. No admin intervention is required. No ticket is needed to process users' requests. Provisioning happens behind the scenes. OpenStack takes care of everything.</p>
              </div>
            </div>
          </li>
          <hr class="p-rule--muted u-hide--large" />
          <li class="p-stepped-list__item">
            <div class="row">
              <div class="col-3 col-medium-3">
                <h3 class="p-heading--5 p-stepped-list__title">OpenStack handles the request</h3>
              </div>
              <div class="col-6 col-medium-3">
                <p class="p-stepped-list__content">Depending on the type of a request, one or another service gets involved in its processing. Each service manages its own domain and provides all necessary functions to handle the given task. This includes user authentication, authorization, resource scheduling, metrics collection, etc.</p>
              </div>
            </div>
          </li>
          <hr class="p-rule--muted u-hide--large" />
          <li class="p-stepped-list__item">
            <div class="row">
              <div class="col-3 col-medium-3">
                <h3 class="p-heading--5 p-stepped-list__title">OpenStack provisions resources</h3>
              </div>
              <div class="col-6 col-medium-3">
                <p class="p-stepped-list__content">In order to provision the actual resource (a VM, a block storage volume, a snapshot or an external IP address), OpenStack relies on other technologies. This includes the <a href="https://ubuntu.com/server/docs/virtualisation-with-qemu">KVM</a> hypervisor, <a href="https://ubuntu.com/blog/data-centre-networking-what-is-ovn">OVN</a> for networking, <a href="https://ubuntu.com/ceph">Ceph</a> for storage and many, many more.</p>
              </div>
            </div>
          </li>
        </ol>
      </div>
```

We can also add this to styles.scss to more closely match the design:
```
.p-stepped-list--detailed {
  .p-stepped-list__title {
    &::before {
        text-align: left;
    }
    padding-left: 2rem;
  }
}
``` 

before (demo):
![image](https://github.com/user-attachments/assets/a02d1869-6e78-4778-b100-0bf6b9f8c87d)


after: 
![image](https://github.com/user-attachments/assets/2d2ed405-ea4f-44da-8d8c-ec7b0af6ffd4)